specification/workloads/resource-manager/Microsoft.Workloads/SAPVirtualInstance/preview/2023-10-01-preview/SAPVirtualInstance.json

```diff
@@ -0,0 +1,4452 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2023-10-01-preview",
+    "title": "Workloads Client",
+    "description": "Workloads client provides access to various workload operations."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Workloads/locations/{location}/sapVirtualInstanceMetadata/default/getSizingRecommendations": {
+      "post": {
+        "tags": [
+          "SAPRecommendations"
+        ],
+        "description": "Get SAP sizing recommendations by providing input SAPS for application tier and memory required for database tier",
+        "operationId": "SAPSizingRecommendations",
```

Comment:
follow https://github.com/Azure/azure-openapi-validator/blob/main/docs/operation-id.md#output-message,
I don't think this operation ID is valid here

Reply:
The operationId seems wired to me, but it is released in previous version. @weidongxu-microsoft how do you think?